KingfisherWebP/Classes/CGImage+WebP.h

```diff
@@ -10,6 +10,7 @@
 
 CF_IMPLICIT_BRIDGING_ENABLED
 
+NSUInteger GetWebPFrameCount(CFDataRef __nullable webpData);
```

Comment:
To follow Apple's suggested naming conventions (like `CFDataGetBytes` and `CGColorSpaceGetColorTableCount`), this function should be renamed something like `WebPDataGetFrameCount`😄